homework/index.js

```diff
@@ -1,25 +1,19 @@
 'use strict';
 
 {
-  function fetchJSON(url, cb) {
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', url);
-    xhr.responseType = 'json';
-    xhr.onload = () => {
-      if (xhr.status >= 200 && xhr.status <= 299) {
-        cb(null, xhr.response);
-      } else {
-        cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
-      }
-    };
-    xhr.onerror = () => cb(new Error('Network request failed'));
-    xhr.send();
+  async function fetchJSON(url) {
+    const response = await fetch(url);
+    if (!response.ok) {
+      throw Error(`Network error: ${response.status} - ${response.statusText}`);
+    }
+    return response.json();
   }
 
   function createAndAppend(name, parent, options = {}) {
     const elem = document.createElement(name);
     parent.appendChild(elem);
-    Object.entries(options).forEach(([key, value]) => {
+    Object.keys(options).forEach(key => {
+      const value = options[key];
```

Comment:
I'm still expecting an answer from you where this code change come from. It is code that we used in previous classes but not in your class.

Reply:
Hi @remarcmij, yes it's a code from previous classes i guess. because when i was trying to do the homework i was need to see the old pull request because i didn't understand the homework correctly. I saw few olds codes and I copied it to my  VS code just to understand it.

Reply:
Of course, I already suspected that. If you don't understand the homework, you should ask in slack, Copying from others, whether it be classmates or homework of previous classes goes against the HYF code of conduct as stated in the Student Manual.

Reply:
Yes but I just was trying to understand one or two point because it will not be efficient if i take the whole code.  i have never did that